dotnet/src/SemanticKernel.Core/Data/KernelBuilderExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.SemanticKernel.Data;
+
+/// <summary>
+/// Extension methods to register Data services on the <see cref="IKernelBuilder"/>.
```

Comment:
Is "Data services" the right term or should we just say "Extension methods to register vector stores on" ...

Reply:
The intent is to have one of these classes in the Data folder, not one for vector stores and more for anything else that may be in the data folder.  I don't expect there to be so many that we would need separate ones.  Hence the name of the class, and the comment.